ckanext/geodatagov/tests/mock_csw_source.py

```diff
@@ -1,18 +1,18 @@
 from __future__ import print_function
 
-import os
 import json
-import re
-import copy
-import urllib
-from urlparse import parse_qs
-import xml.etree.ElementTree as ET
+import logging
+import os
 import SimpleHTTPServer
 import SocketServer
+import xml.etree.ElementTree as ET
 from threading import Thread
-import logging
-log = logging.getLogger("harvester")
+from urlparse import parse_qs
 
+from ckanext import geodatagov
+
+log = logging.getLogger("harvester")
+SAMPLES_PATH = os.path.join(geodatagov.__path__[0], 'tests/data-samples')
```

Comment:
I think the preferred way to do this is `dirname(geodatagov.__file__)` but my python is a bit rusty.

Reply:
And really, I think there as an even better way with module resources but I'm not super familiar with that.

Reply:
I can't use the file method here because geodatagov is being declared as a namespace in the [init](https://github.com/thejuliekramer/ckanext-geodatagov/blob/master/ckanext/__init__.py) file. I can remove this configuration in a separate PR and test if that makes sense?